azerothcore/azerothcore-wotlk
# fix(Docker): allow to use modules

##### CHANGES PROPOSED:

-  Include modules in Docker build

##### TESTS PERFORMED:

Not yet tested.

##### HOW TO TEST THE CHANGES:

Include a module in your sources as you would normally do in a traditional setup. Then run the docker setup and see if the module is correctly included.


##### Target branch(es):

Master
